v2g-liberty-cards/src/util/utils.ts

```diff
@@ -0,0 +1,14 @@
+export function elapsedTimeSince(dateTimeStamp: string) {
```

Comment:
Suggestion: instead of creating one utils with a number of functions, consider creating smaller more dedicated files. E.g. here `time.ts` would be a good choice for all time related functions.

Reply:
OK, will do in next PR.

Reply:
Why not update this PR instead?

Reply:
Will do, what about utils_time.ts?